### ISSUE TITLE
invalid-meta

### ISSUE THREAD

**Issue by ajsb85** [choffmeister/roboto-fontface-bower#35]
```sh
bower roboto-fontface#*   invalid-meta for:\bower.json
bower roboto-fontface#*   invalid-meta The "main" field cannot contain globs (example: "*.js")
bower roboto-fontface#*   invalid-meta The "main" field cannot contain globs (example: "*.js")
bower roboto-fontface#*   invalid-meta The "main" field has to contain only 1 file per filetype; found multiple .css files: ["./css/roboto/roboto-fontface.css","./css/roboto-condensed/r
oboto-condensed-fontface.css"]
bower roboto-fontface#*   invalid-meta The "main" field has to contain only 1 file per filetype; found multiple .* files: ["./fonts/roboto/*.*","./fonts/roboto-condensed/*.*"]
```

**Comment by choffmeister** [comment]
@ajsb85 I don't use bower anymore since a long time. Can you support me in fixing this? I don't know what the correct and most common way to populate the `main` field for bower is.

**Comment by ajsb85** [comment]
_From bower.json specification_

### main

*Recommended*

Type: `String` or `Array` of `String`

The entry-point files necessary to use your package. Only one file per filetype.

Entry-point files have module exports and may use module imports. While Bower does not directly use `main` files, they are listed with the commands `bower list --json` and `bower list --paths`, so they can be used by build tools.

Let's say your package looks like this:

    package
      js/
        motion.js
        run.js
        walk.js
      sass/
        motion.scss
        run.scss
        walk.scss
      img/
        motion.png
        walk.png
        run.png
      fonts/
        icons.woff2
        icons.woff
      dist/
        movement.js
        movement.min.js
        movement.css
        movement.min.css

`motion.js` has module imports for `run.js` and `walk.js`. `motion.scss` has module imports for `run.scss` and `walk.scss`. `main` would be

```
"main": [
  "js/motion.js",
  "sass/motion.scss",
]
```

Image and font files may be used or referenced within the JS or Sass files, but are not `main` files as they are not entry-points.

* Use source files with module exports and imports over pre-built distribution files.
* Do not include minified files.
* Do not include assets files like images, fonts, audio, or video
* Filenames should not be versioned (Bad: `package.1.1.0.js`; Good: `package.js`).
* Globs like `js/*.js` are not allowed.

**Comment by choffmeister** [comment]
@ajsb85 Thx for the fix. Released as 0.6.1
